docs/DeveloperGuide.adoc

```diff
@@ -769,12 +825,90 @@ Use case resumes at step 2.
 
 {More to be added}
 
+[discrete]
+=== Use case: Sort contacts alphabetically
+
+*MSS*
+
+1. User requests to sort contacts alphabetically
+2. AddressBook sorts contacts alphabetically according to their names
+3. AddressBook shows contact list to user
++
+Use case ends.
+
+*Extensions*
+
+[none]
+* 1a. The list is empty.
++
+[none]
+** 1a1. AddressBook shows “empty list of contacts” to user
++
+Use case ends.
+
+[discrete]
+=== Use case:  Adding a tag to a person
+
+*MSS*
+
+1. User requests to list persons
+2. AddressBook shows list of persons
+3. User requests to add a specific tag to a person in the list
+4. AddressBook confirms the details of the instruction with the user
+5. User confirms command
+6. AddressBook adds tag to this person
++
+Use case ends
+
+*Extensions*
+
+[none]
+* 1a. The list is empty.
++
+[none]
+** 1a1. AddressBook shows “empty list of contacts” to user
```

Comment:
you can end your use case without this since you're not going to interact with `AddressBook` any further.